ckanext/validation/jobs.py

```diff
@@ -117,18 +126,21 @@ def run_validation_job(resource):
          'validation_timestamp': validation.finished.isoformat()})
 
 
-def _validate_table(source, _format=u'csv', schema=None, **options):
+def _validate_table(source, _format='csv', schema=None, **options):
 
+    frictionless_context = { 'trusted': True }
```

Comment:
I could not find any documentation in the fd v5 docs for what this property does, can you explain it?

Reply:
This is necessary when you pass a full path to the validate function, otherwise it fails. I did need a hard time to discover that because it isn't in the docs. This option is there to follow the standard https://specs.frictionlessdata.io/data-resource/#data-location that discourages the use of absolute paths

Reply:
Cool, let's add a comment then to clarify this
```suggestion
    # This option is needed to allow Frictionless Framework to validate absolute paths
    frictionless_context = { 'trusted': True }
```